.circleci/config.yml

```diff
@@ -22,55 +27,14 @@ jobs:
       - run: sudo -E kubectl cluster-info
       - restore_cache:
           keys:
-            - bazel-cache-{{ checksum "WORKSPACE" }}
-      - run: make build
-      - save_cache:
-          key: bazel-cache-{{ checksum "WORKSPACE" }}
-          paths:
-            - /home/circleci/.cache/bazel      
-      - run: make docker HUB=$HUB TAG=$TAG  # make docker images for everything
-      - run: cd pilot; mkdir /home/circleci/logs; make e2etest HUB=$HUB TAG=$TAG TESTOPTS="-mixer=false -errorlogsdir=/home/circleci/logs"
+            - dep-cache-{{ checksum "Gopkg.lock" }}
+      - run: bin/purego-build-pilot -hub $HUB -tag $TAG -build-only
+      - run: mkdir /home/circleci/logs
+      - run: go build -i ./pilot/test/integration
+      - run: ./integration --logtostderr -hub $HUB -tag $TAG -mixer=false -errorlogsdir=/home/circleci/logs 
       - store_artifacts:
           path: /home/circleci/logs
 
-  unittest:
```

Comment:
are you removing `unittest` or did you just remove it temporarily to make experimentation faster?

Reply:
temporary.. fixed